test/_is_cassandra_incompat.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict';
+
+var semver = require('semver');
+const { getLockedVersion } = require('./_utils');
+
+/**
+ * Return whether the current 'cassandra-driver' version is incompatible with the
+ * current version of node. If so, this returns a string that can be used
+ * as a reason for a test skip message. Otherwise it returns false.
+ *
+ * Usage:
+ *    const isCassandraIncompat = require('.../_is_cassandra_incompat')()
+ *    if (isCassandraIncompat) {
+ *      console.log(`# SKIP ${isCassandraIncompat}`)
+ *      process.exit()
+ *    }
+ *
+ * Dev Note: These version ranges should mirror the cassandra section of ".tav.yml".
+ *
+ * @returns {string | boolean}
+ */
+function isCassandraIncompat() {
+  const nodeVer = process.version;
+  const cassandraVer = getLockedVersion('cassandra-driver');
```

Comment:
I don't understand why using `getLockedVersion` is necessary or wanted vs. using `safeGetPackageVersion`.  `safeGetPackageVersion` will get the currently installed package version without `require`ing it.

As well, getting the version from the lock file will *not* necessarily return the currently installed version when TAV is being used, because it does `npm install --no-save PKG@VERSION` to not update the local package.json and package-lock.json when it is working through versions.

Reply:
You're right. Error came from another place and this stayed. Going to remove and use `safeGetPackageVersion`

Reply:
removed!